### PR TITLE
DOCSP-40539 Add Airbnb project to Example Projects

### PR DESCRIPTION
**Pull request by lindseymoore**
## Pull Request Info - SDK Docs Consolidation

Jira ticket: https://jira.mongodb.org/browse/DOCSP-40539

*Staged Page*

- [Example Projects](https://preview-mongodblindseymoore.gatsbyjs.io/realm/DOCSP-40539/example-projects/)

### PR Author Checklist

Before requesting a review for your PR, please check these items:

- [x] Tag the consolidated page for:
  - genre
  - meta.keywords
  - meta.description

### CODE REVIEW

File: source/example-projects.txt [hunk 1]
```diff
@@ -192,6 +192,12 @@ Check out these supplementary projects to build on the power of the Atlas Device
       Use multiple realms to allow registered users to sync and manage movies in their own private list.
     - - `React Native <https://github.com/realm/realm-js/tree/main/examples/rn-multiple-realms>`__
 
+  * - Searchable Offline Collection
+    - Build an application that caches viewed collection items, so you can search and view them offline. 
+      This demo application uses listings from the
+      `Sample AirBnB Listings Dataset <https://www.mongodb.com/docs/atlas/sample-data/sample-airbnb/>`__.
```
Comment (cbullinger) [inline source/example-projects.txt:198]: not super important, but I noticed referred to a previous sample dataset a little differently (we refer to it as `MongoDB's <Mflix> sample dataset`).

File: source/example-projects.txt [hunk 2]
```diff
@@ -192,6 +192,12 @@ Check out these supplementary projects to build on the power of the Atlas Device
       Use multiple realms to allow registered users to sync and manage movies in their own private list.
     - - `React Native <https://github.com/realm/realm-js/tree/main/examples/rn-multiple-realms>`__
 
+  * - Searchable Offline Collection
+    - Build an application that caches viewed collection items, so you can search and view them offline. 
+      This demo application uses listings from the
```
Comment (cbullinger) [inline source/example-projects.txt:197]: [suggestion] since we're tight on space
```suggestion
      This demo app uses listings from the
```

**Review by cbullinger — APPROVED** [review]
lgtm!

File: source/example-projects.txt (hunk 1 above)
Comment (lindseymoore) [reply source/example-projects.txt:198]: I'll keep the naming consistent with what's on the dataset page (though the project itself is marketed as MBnB).

**Pull request merged**